modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/CSharpClientCodegen.java

```diff
@@ -153,8 +194,6 @@ public void processOpts() {
         supportingFiles.add(new SupportingFile("ApiResponse.mustache",
                 clientPackageDir, "ApiResponse.cs"));
 
-        supportingFiles.add(new SupportingFile("Newtonsoft.Json.dll", "bin", "Newtonsoft.Json.dll"));
-        supportingFiles.add(new SupportingFile("RestSharp.dll", "bin", "RestSharp.dll"));
```

Comment:
I can lock restsharp version down to 105.1.0, which is what was provided previously, then it should still work as expected via nuget.

Do you think I should attempt to restructure the project output as part of this pull?